### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by rakleed** [xenia-project/xenia#2149]
It would be nice to have a dark theme in the Xenia. Because using it at night or in a dark room is very unpleasant because of the bright white background. Ideally, it would be good if the theme of Xenia matches the theme of the system.

**Comment by Triang3l** [comment]
The contents of the window on launch are black now, it looks like you're using some extremely outdated (pre-early-2022) version. However, there's nothing to apply themes to so far, as the UI currently only contains a thin menu bar in the top, which is a standard Win32 menu. `DWMWA_USE_IMMERSIVE_DARK_MODE`, unfortunately, only controls the color of the title bar, but not the menu.

**Comment by rakleed** [comment]
@Triang3l yeah, but still, the background where the game starts is not the whole application interface. Therefore, my request refers specifically to the title bar and menu items.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/19418601/232820865-82125f3e-7c79-4381-800c-2a6a8238a0aa.png)

</details>

What do you think about the dark theme implementation of Notepad++? https://github.com/notepad-plus-plus/notepad-plus-plus/commit/5a3bf49c86acca8863a1be2fb12038f18c54594a

**Comment by Triang3l** [comment]
If the dark mode requires hard-coding any specific RGB colors in Xenia, I don't think it'd be the right way to go (not friendly to system configuration, to the user's Windows theme) when using the OS's controls if they don't support the dark mode internally.

Though anyway, this whole GUI is slooowly being replaced with a completely new frontend, which is dark by default, so it's on life support overall.
